prerendering.bs

```diff
@@ -658,6 +673,33 @@ The <dfn attribute for="PerformanceNavigationTiming">activationStart</dfn> gette
 
 1. Return a {{DOMHighResTimeStamp}} with a time value equal to the [=current document=]'s [=Document/activation start time=].
 
+<h3 id="client hint cache">Interaction with Client Hint Cache</h3>
+
+[=Accept-CH cache=], which is owned by user agent as a global storage, should not be modified by [=prerendering browsing context=].
+
+Instead, each [=prerendering browsing context=] maintains its <dfn>prerender scoped Accept-CH cache</dfn>, to persist the origin's opt-in to receive the indicated header fields. 
```

Comment:
Maybe this should be a `<dfn for="prerendering browsing context">`? Also, what type of value is this?

Reply:
Thanks! I updated its type. 

(BTW: Sorry for the stupid question. I'd like to understand when do we usually use "for=xxx". I guess we use "for" when a property belongs to another? It might be similar to a class member from programming's perspective. Is my understanding correct?

Reply:
Yes, your understanding is correct!

One detail is that it's not always mandatory in spec-writing, since the for="" value is only used for disambiguation. But I think it's always good practice to include it, even if there is no other type of object that has a property named "prerender scoped Accept-CH cache". It makes it clearer to people reading the spec source code, and protects you from any future collision.

Reply:
Thank you, Domenic! 

I see. Yeah, when I read the specification, I find it is very helpful! I'll be careful when I write specificaiotn :D